capstone-app/src/app/modules/history/services/wiki-search-handler.service.ts

```diff
@@ -8,7 +8,7 @@ import { catchError, map } from 'rxjs/operators';
 })
 export class WikiSearchHandler {
   private static LINK = 'https://www.googleapis.com/customsearch/v1?';
-  private static API_KEY = 'key=AIzaSyCd2vIkDcVuvM3ie4kE661cGS3GsH9IWcQ';
+  private static API_KEY = 'key=AIzaSyBdi-mLi0sRpQT3z9T0aKfEfuTRqAhZVYQ';
```

Comment:
Just wanted to double-check we're not exposing the "good" api key.

Reply:
I accidently did push the good key a few days ago but immediately deleted it and regenerated. So this is the 'old'  good key